src/app/core/data/item-data.service.ts

```diff
@@ -118,4 +121,34 @@ export class ItemDataService extends DataService<Item> {
       map((requestEntry: RequestEntry) => requestEntry.response)
     );
   }
+
+  public getMoveItemEndpoint(itemId: string): Observable<string> {
```

Comment:
Please add TypeDocs for this public method.

Reply:
Added typedocs